tests/test_rdscluster.py

```diff
@@ -512,3 +512,118 @@ def test_rdscluster_snapshot_trim(self):
         )
         resources = p.run()
         self.assertEqual(len(resources), 2)
+
+    def _get_effective_permissions(self, client, snapshot_id):
+        attributes = client.describe_db_cluster_snapshot_attributes(
+            DBClusterSnapshotIdentifier=snapshot_id
+        )["DBClusterSnapshotAttributesResult"]["DBClusterSnapshotAttributes"]
+        attr_map = {
+            attr["AttributeName"]: attr["AttributeValues"]
+            for attr in attributes
+        }
+        return set(attr_map.get("restore", []))
+
+    def test_set_permissions(self):
+        session_factory = self.replay_flight_data(
+            "test_rdscluster_set_permissions",
+            region="us-east-2"
+        )
+        target_snapshot_id = "test"
+        keep = "644160558196"
+        remove = "123456789012"
+        add = "234567890123"
+        policy = self.load_policy(
+            {
+                "name": "rds-snapshot-remove-cross-account",
+                "resource": "rds-cluster-snapshot",
+                "filters": [
```

Comment:
use config to select the resource there are a number of extraneous snapshots in the describe output here which are bloating this pr with extraneous recorded data.

Reply:
Ok done - while making that change I hit errors with tag lookups for the cluster snapshot config source. Resolved in 19a2f3f - seemed straightforward but worth pointing out in case I'm missing something.